t/test.t

```diff
@@ -63,7 +63,7 @@ my $critic = Perl::Critic->new(
 
     my @violations = $critic->critique( \$str );
 
-    is( scalar @violations, 0 );
+    is( scalar @violations, 0, "No bug - with or and parenthesis");
```

Comment:
Please use single-quotes for non-interpolated strings as described in [ProhibitInterpolationOfLiterals](https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals)

```suggestion
    is( scalar @violations, 0, 'No bug - with or and parenthesis' );
```